packages/amplify-cli-core/src/customPoliciesUtils.ts

```diff
@@ -0,0 +1,57 @@
+import { Fn, IAM } from "cloudform-types";
+
+export type CustomIAMPolicies = {
+    policies:  CustomIAMPolicy[];
+}
+
+export type CustomIAMPolicy = {
+    Action: string[];
+    Effect: string;
+    Resource: string[];
+}
+
+
+export const CustomIAMPoliciesSchema = {
+    type : "object",
+    properties: {
+        policies: {type: "array", minItems: 1, items: {type: "object"}}
+    },
+    required: ["policies"],
+    additionalProperties: false
+}
+
+export const CustomIAMPolicySchema = {
+        type: "object",
+        properties: {
+            Action: {type: "array", items: {type: "string"}, nullable: false},
+            Effect: {type: "string", items: {type: "string"}, nullable: true, default: "Allow"},
+            Resource: {type: "array", items: {type: "string"}, minItems: 1, nullable: false},
+        },
+        required: ["Resource", "Action"],
+        additionalProperties: false
+}
+
+export const customExecutionPolicyForFunction = new IAM.Policy({
+    PolicyName: 'custom-lambda-execution-policy',
```

Comment:
Casing is different

Reply:
The casing is following the cfn template's casing